go.mod

```
@@ -10,7 +10,7 @@ require (
 	github.com/celestiaorg/go-libp2p-messenger v0.1.0
 	github.com/celestiaorg/nmt v0.8.0
 	github.com/celestiaorg/rsmt2d v0.4.0
-	github.com/cosmos/cosmos-sdk v0.44.3
+	github.com/cosmos/cosmos-sdk v0.46.0
```

As I see in cosmos-sdk tag, they don't have v0.46.0 rather then beta-2
```suggestion
	github.com/cosmos/cosmos-sdk v0.46.0-beta2
```
https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.0-beta2